src/main/resources/public/js/graphs.controller.js

```diff
@@ -73,7 +73,7 @@
         return;
       }
 
-      const sortedActivities = _.sortBy(activities, CONST.DATE);
+      _.sortBy(activities, CONST.DATE);
```

Comment:
@sepe81 This fixes the graph bug. It is possible that the sorted array should not be assigned by _.sortBy.  I changed this to `sortedActivities` because of the rule `no-param-reassign` and added the `const` keyword.